pkg/progress/progress_watcher.go

```diff
@@ -0,0 +1,124 @@
+package progress
```

Comment:
It's better to put license as header If it's going to become Eclipse project. 
See how to automate it for go here https://github.com/devfile/devworkspace-operator/blob/master/Makefile

Reply:
Thanks :)
Add license header manually. Automate checking will be added in separate PR.